packages/skill-api/src/core/services/convertkit/convertkit-tag-purchase.ts

```diff
@@ -46,57 +44,62 @@ async function getSanityProduct(productId: string) {
 
 export async function convertkitTagPurchase(email: string, purchase: Purchase) {
   // skip CK tagging in development
-  if (process.env.SKIP_CK_TAGGING === 'true') return
+  if (process.env.SKIP_CK_TAGGING === 'true')
+    return {tagged: false, reason: 'SKIP_CK_TAGGING is true'}
```

Comment:
there were a mix of boolean and `{ status: 200 }` return types from this function, trying to unifying into something more consistent.

(though it doesn't look like the call site is using the return type)